src/Uno.ReferenceImplComparer/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Transactions;
+using Mono.Cecil;
+using Mono.Collections.Generic;
+
+namespace Uno.ReferenceImplComparer
+{
+	class Program
+	{
+		static int Main(string[] args)
+		{
+			var hasErrors = false;
+			var filePath = UnpackArchive(args[0]);
+
+			Console.WriteLine($"Validating package {args[0]}");
+
+			foreach(var assembly in Directory.GetFiles(Path.Combine(filePath, "lib", "netstandard2.0"), "*.dll"))
+			{
+				var referenceAssemblyDefinition = ReadAssemblyDefinition(assembly);
+
+				foreach(var runtimeAssembly in Directory.GetFiles(Path.Combine(filePath, "uno-runtime"), Path.GetFileName(assembly), SearchOption.AllDirectories))
+				{
+					var identifier = $"{Path.GetFileName(runtimeAssembly)}/{Path.GetFileName(Path.GetDirectoryName(runtimeAssembly))}";
+					Console.WriteLine($"Validating {identifier}");
+
+					var runtimeAssemblyDefinition = ReadAssemblyDefinition(runtimeAssembly);
+
+					hasErrors |= CompareAssemblies(referenceAssemblyDefinition, runtimeAssemblyDefinition, identifier);
+				}
+			}
+
+			return hasErrors ? 1 : 0;
+		}
+
+		private static bool CompareAssemblies(AssemblyDefinition referenceAssemby, AssemblyDefinition runtimeAssembly, string identifier)
+		{
+			var hasError = false;
+			var referenceTypes = referenceAssemby.MainModule.GetTypes();
+			var runtimeTypes = runtimeAssembly.MainModule.GetTypes().ToDictionary(t => t.FullName);
+
+			foreach(var referenceType in referenceTypes.Where(t => t.IsPublic))
+			{
+				if(referenceType.FullName == "Windows.UI.Xaml.Documents.TextElement")
+				{
+					Console.WriteLine("Skipping Windows.UI.Xaml.Documents.TextElement comparison");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)